.github/workflows/potential-duplicates.yml

```diff
@@ -0,0 +1,36 @@
+name: Potential Duplicates
+on:
+  issues:
+    types: [opened, edited]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: bubkoo/potential-duplicates@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # Label to set, when potential duplicates are detected.
+          label: potential duplicate
+          # Exclude keyworlds in title before detecting.
+          exclude: |
+            icon
+            icons
+            logo
+            logos
+            request
+            request:
+            add
+            adding
+          # Get issues with state to compare. Supported state: 'all', 'closed', 'open'.
+          state: all
+          # If similarity is higher than this threshold([0,1]), issue will be marked as duplicate.
+          threshold: 0.7
+          # Reactions to be add to comment when potential duplicates are detected.
+          # Available reactions: "-1", "+1", "confused", "laugh", "heart", "hooray", "rocket", "eyes"
+          reactions: eyes
+          # Comment to post when potential duplicates are detected.
+          comment: >
```

Comment:
If I'm not mistaken, the `>` here tells YAML to remove the newlines in the comment template below, but I would expect it to keep them in which case I think this should be `|`.

(see https://yaml-multiline.info/)

Reply:
Hum... Good question!
Newlines are kept right now using the current configuration as seen [here](https://github.com/service-paradis/service-paradis/issues/10)
![image](https://user-images.githubusercontent.com/12817388/106755918-ae938700-65fc-11eb-8093-843fb48ba4ca.png)

> Lines with extra indentation are also not folded.

I think this is the reason why newlines are kept right now. I can try it out with literal style (`|`) to see if it works the same way!

Reply:
I can confirm that using the pipe gives the same expected results. Will use it!
![image](https://user-images.githubusercontent.com/12817388/106757131-fd8dec00-65fd-11eb-994f-cddd1f2db063.png)